hack/ci/e2e-conformance.sh

```diff
@@ -321,6 +321,8 @@ EOF
   source "${REPO_ROOT}/hack/ensure-go.sh"
   source "${REPO_ROOT}/hack/ensure-kind.sh"
 
+  gcloud components update --quiet
```

Comment:
This might not be a great idea in terms of making the test reproducible.  I'm guessing this is a temporary workaround until we can get the upstream e2e image fixed up?  If so, can we have a comment to that effect?

Reply:
oops sorry forgot to take it out.